Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM quay.io/powerhome/passenger-customizable:0.9.35
+FROM phusion/passenger-customizable:1.0.19
```

Comment:
I'm wondering if we might encounter rate limiting issues by using this image rather than a `1.0.19` image we host on quay.io. 

I'm mentioning this because I noticed this note on the description of the `quay.io/powerhome/passenger-customizable:0.9.35` image on [quay.io](https://quay.io/repository/powerhome/passenger-customizable):

> Mirror of phusion/passenger-customizable under our control.
> https://www.docker.com/increase-rate-limits screws us otherwise.

Perhaps this image is fine to use so we can become unblocked, and then we'll want to change to a Quay.io hosted image soon?

Reply:
That's basically what https://nitro.powerhrg.com/runway/backlog_items/RBLS-1424 is about. It's not important for this PR, but it would be nice to add this latest version of the image to Quay.